app/services/stats/compute_rate_of_applicants_with_rdv_seen_posterior_to_an_invitation.rb

```diff
@@ -0,0 +1,36 @@
+module Stats
+  class ComputeRateOfApplicantsWithRdvSeenPosteriorToAnInvitation < BaseService
+    def initialize(invitations:)
+      @invitations = invitations
+    end
+
+    def call
+      result.value = compute_rate_of_applicants_with_rdv_seen_posterior_to_an_invitation
+    end
+
+    private
+
+    def compute_rate_of_applicants_with_rdv_seen_posterior_to_an_invitation
+      (applicants_with_rdv_seen_posterior_to_an_invitation.count / (
+        applicants.count.nonzero? || 1
+      ).to_f) * 100
+    end
+
+    def invitations_that_led_to_a_rdv_seen
+      @invitations.select do |invitation|
+        invitation.participations.seen.any? do |participation|
+          participation.created_at > invitation.created_at
+        end
+      end
```

Comment:
On pourrait pas utiliser une requête sql pour faire ça ? Ça risque de faire beaucoup d'itérer sur toutes les invitations

Reply:
je n'ai pas trouvé de moyen, puisqu'il faut vérifier pour chaque invitation si elle répond aux critères ou non. On pourrait récupérer la partie relative à `invitation.participations.seen.any?` avec du SQL/Active record, mais pas la deuxième partie (enfin en tout cas je n'ai pas trouvé comment). C'est d'ailleurs en grande partie pour ça que j'en ai fait un service différent du service pour les stats globales (où l'on a pas besoin de faire aussi compliqué puisque une participation `seen` suffit à valider qu'il y a eu une orientation)